web/src/components/Link.tsx

```diff
@@ -0,0 +1,20 @@
+export function Link({
+  href,
+  rel,
+  children,
+}: {
+  href: string;
+  rel?: string;
+  children: React.ReactNode;
+}) {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel={`noopener ${rel}`}
```

Comment:
It's currently returning `undefined` and I don't think we need to make this dynamic :)
```suggestion
      rel="noopener"
```